Source/Scene/Model.js

```diff
@@ -1654,67 +1810,74 @@ define([
 
     // This doesn't support LOCAL, which we could add if it is ever used.
     var gltfSemanticUniforms = {
-        MODEL : function(uniformState) {
+        MODEL : function(uniformState, model) {
             return function() {
                 return uniformState.model;
             };
         },
-        VIEW : function(uniformState) {
+        VIEW : function(uniformState, model) {
             return function() {
                 return uniformState.view;
             };
         },
-        PROJECTION : function(uniformState) {
+        PROJECTION : function(uniformState, model) {
             return function() {
                 return uniformState.projection;
             };
         },
-        MODELVIEW : function(uniformState) {
+        MODELVIEW : function(uniformState, model) {
             return function() {
                 return uniformState.modelView;
             };
         },
-        MODELVIEWPROJECTION : function(uniformState) {
+        CESIUM_RTC_MODELVIEW : function(uniformState, model) {
+            // CESIUM_RTC extension
+            var mvRtc = new Matrix4();
```

Comment:
Does this end up being a per-model scratch?  How often is `CESIUM_RTC_MODELVIEW` called?